packages/utilities/gel-foundations-styled-components/typography_sizes_web.md

```diff
@@ -0,0 +1,23 @@
+ # GEL Typography Sizes (em/rem)
+ 
+These values are based on the [GEL Typography guidelines](https://www.bbc.co.uk/gel/guidelines/typography#type-sizes), which are defined in pixels. The values below can be recreated and verified by simply dividing the pixel values by 16 (16px is the base font size for the majority of browsers).
+
+Note that this library does not implement the "Group C" sizes. This group is intended for large touch devices, and is not currently anticipated to be required for users of this library.
+
+||Group A Font Size|Group A Line Height|Group B Font Size|Group B Line Height|Group C Font Size|Group C Line Height|Group D Font Size|Group D Line Height|
+|---|-----------------|-------------------|-----------------|-------------------|-----------------|-------------------|-----------------|-------------------|
```

Comment:
Can we make this a little nicer formatted?

```markdown
| -------- | ----- | ---- | ----  | ---- | ---- | ----   | ---- | ---- |
| Atlas    | 4.875 | 5.25 | 6     | 6.5  | 12   | 13     | 8.75 | 9.25 |
| Elephant | 3.75  | 4    | 4.875 | 5.25 | 9.75 | 10.625 | 7.25 | 7.75 |
```

Reply:
I'll have a go. Thank you very much for the thorough review! 🙌🏻